src/webextension/ext-main.c

```diff
@@ -84,6 +84,9 @@ static const char introspection_xml[] =
     "  <method name='SetHeaderSetting'>"
     "   <arg type='s' name='headers' direction='in'/>"
     "  </method>"
+    "  <method name='FocusElementById'>"
+    "   <arg type='s' name='elemend_id' direction='in'/>"
```

Comment:
Parameter for the pageId is required too `<arg type='t' name='page_id' direction='in'/>`.